### DISCUSSION TITLE
nocodb timezone [Oct 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4273] [Discord Threads]
> 🙋‍♂️ **ofir**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### are you talking about the docker container timezone? if so, you can specify it in `TZ` .   
####    
#### e.g.   
####    
#### ```   
#### docker run -d -e TZ=Asia/Hong_Kong --name nocodb -p 8080:8080 nocodb/nocodb:latest   
#### ```  
#  
> 🙋‍♂️ **ofir**  
#### no... in npm install.   
#### on formula NOW()   
#### i need geting my timezone.  
#  
> 🙋‍♂️ **ofir**  
#### ?  
#  
> 🙋‍♂️ **wkw**  
#### i think that s reading your db timezone  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1032193380194734140)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
